test/RecipeRepository-test.js

```diff
@@ -48,28 +48,42 @@ describe('RecipeRepository', () => {
     expect(recipesByTag).to.deep.equal([sampleData[0], sampleData[2], sampleData[7]]);
   });
 
-  it('should be able to take in an ingredient name and return an array containing that ingredients ID', () => {
-    const getIngID = repository.getIngredientID('wheat flour', ingredientsData);
+  it('should be able to take in an ingredient name and return the corresponding ID for that ingredient', () => {
+    const ingID = repository.getIngredientID('wheat flour', ingredientsData);
 
-    expect(getIngID).to.deep.equal([20081])
+    expect(ingID).to.deep.equal(20081)
   });
 
   it('should return a list of recipes based on its name', () => {
-    const recipesByName = repository.filterByNameOrIng('Loaded Chocolate Chip Pudding Cookie Cups', ingredientsData);
-    let recipe1 = sampleData[0];
-    let recipesTest = [recipe1];
+    let recipesByName = repository.filterByName('Loaded Chocolate Chip Pudding Cookie Cups', ingredientsData);
+
+    let recipesTest = [sampleData[0]];
 
     expect(recipesByName).to.deep.equal(recipesTest);
   });
 
   it('should return a list of recipes based on its ingredient', () => {
-    const recipesByIngredient = repository.filterByNameOrIng('brown sugar', ingredientsData);
+    let recipesByIng = repository.filterByIng('brown sugar', ingredientsData);
+
     let recipe1 = sampleData[0];
     let recipe2 = sampleData[2];
     let recipe3 = sampleData[6];
     let recipe4 = sampleData[8];
+
     let recipesTest = [recipe1, recipe2, recipe3, recipe4];
 
-    expect(recipesByIngredient).to.deep.equal(recipesTest);
+    expect(recipesByIng).to.deep.equal(recipesTest);
   });
+
+  // these test to make sure we are not adding duplicates, we can choose one I just wanted to try multiple cases
+  it('should be able to filter by name or ingredient and store result', () => {
```

Comment:
I feel like multiple cases is good. To 'prove' that the method will prevent duplicates.